.github/workflows/compile_protos.yml

```diff
@@ -64,16 +54,19 @@ jobs:
         if: startsWith(env.commit-message, env.CI_COMMIT_MESSAGE_PREFIX) && env.commit-author == env.CI_COMMIT_AUTHOR
         run: echo "is-auto-commit=true" >> $GITHUB_ENV
 
+      - uses: jdx/mise-action@v2
```

Comment:
can you pin mise-action to a sha? unpinned 3rd party actions are a security issue

Reply:
Sure. I'll move the buf action to use a sha while I'm at it unless there's a reason not to. The only reason I went with the version instead of a sha here is because that's how the other 3rd party actions (buf and arduino/setup-protoc) were being used.